README.md

```diff
@@ -206,11 +207,12 @@ The `NuxtAuthHandler` accepts [all options that NextAuth.js accepts for its API
 
 ##### Example with two providers
 
-Here's what a full config can look like, wee allow authentication via a:
-- Github Oauth flow,
+Here's what a full config can look like, we allow authentication via:
+- a Github Oauth flow
 - a username + password flow (called `CredentialsProvider`)
 
-Note that the below implementation of the credentials provider is flawd and mostly copied over from the [NextAuth.js credentials example](https://next-auth.js.org/configuration/providers/credentials) in order to give a picture of how to get started with the credentials provider:
+
+Note that the below implementation of the credentials provider is flawed and mostly copied over from the [NextAuth.js credentials example](https://next-auth.js.org/configuration/providers/credentials) in order to give a picture of how to get started with the credentials provider:
```

Comment:
Let's remove this from here now, as we have it below now, we don't want to double stuff (: